models/WebsiteSetting/Listing.php

```diff
@@ -16,31 +16,32 @@
 namespace Pimcore\Model\WebsiteSetting;
 
 use Pimcore\Model;
+use Pimcore\Model\WebsiteSetting;
 
 /**
  * @method \Pimcore\Model\WebsiteSetting\Listing\Dao getDao()
```

Comment:
Could be changed, too, as `WebsiteSetting` is imported now.

```suggestion
 * @method WebsiteSetting\Listing\Dao getDao()
```

Reply:
yes, thanks!